modules/installation-configuration-parameters.adoc

```diff
@@ -1271,9 +1276,77 @@ Optional VMware vSphere machine pool configuration parameters are described in t
 
 endif::vsphere,vmc[]
 
+ifdef::alibaba[]
+//From: https://github.com/openshift/installer/blob/master/data/data/install.openshift.io_installconfigs.yaml#L20; https://github.com/openshift/openshift-docs/pull/40651/files#r792388476
+
+[id="installation-configuration-parameters-additional-alibaba_{context}"]
+== Additional {alibaba} configuration parameters
+
+Additional {alibaba} configuration parameters are described in the following table:
+
```

Comment:
@mtulio @jianli-wei @kwoodson @joelddiaz
Can someone verify if I got these Alibaba install-config parameters correct?

Reply:
Here is one from the example above installer:
```
apiVersion: v1
baseDomain: alicloud-dev.devcluster.openshift.com
compute:
- architecture: amd64
  hyperthreading: Enabled
  name: worker
  platform: {}
  replicas: 3
controlPlane:
  architecture: amd64
  hyperthreading: Enabled
  name: master
  platform: {}
  replicas: 3
metadata:
  creationTimestamp: null
  name: demo
networking:
  clusterNetwork:
  - cidr: 10.128.0.0/14
    hostPrefix: 23
  machineNetwork:
  - cidr: 10.0.0.0/16
  networkType: OpenShiftSDN
  serviceNetwork:
  - 172.30.0.0/16
platform:
  alibabacloud:
    region: us-east-1
publish: External
pullSecret: '{"auths": {"cloud.openshift.com": {"auth": ... }'
sshKey: |
  ssh-rsa AAAA...
```

Reply:
I looked through these parameters and things look okay except the systemDiskSize is missing. (https://github.com/openshift/installer/blob/master/pkg/types/alibabacloud/platform.go)